README.md

```diff
@@ -72,3 +72,184 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
+
+# MongoDb source for Knative
```

Comment:
seems weird to have this under the development bits. 
It seems that we can repurpose this README to only contain the example, and we can add a reference to a DEVELOPMENT.md at the bottom, pointing the first part of this current README (which we will move to DEVELOPMENT.md). Does it make sense?